specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2018-07-15-preview/definitions/Common.json

```diff
@@ -290,16 +296,73 @@
           "type": "boolean",
           "description": "Whether to trust the server certificate",
           "default": false
+        },
+        "platform": {
+          "$ref": "#/definitions/SqlServerSourcePlatform",
+          "description":  "Server platform type for connection"
         }
       },
-      "required": [ "dataSource" ]
+      "required": [
+        "dataSource"
+      ]
+    },
+    "SqlServerSourcePlatform": {
+      "type": "string",
+      "description": "An enumeration of source platform types",
+      "enum": [
+        "SqlOnPrem"
+      ],
+      "x-ms-enum": {
+        "name": "SqlSourcePlatform",
+        "modelAsString":  true
+      }
+    },
+    "MySqlConnectionInfo": {
+      "x-ms-discriminator-value": "MySqlConnectionInfo",
+      "type": "object",
+      "description": "Information for connecting to MySQL source",
+      "properties": {
+        "serverName": {
+          "type": "string",
+          "description": "Name of the server"
+        },
+        "port": {
+          "type": "integer",
+          "description": "Port for Server"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ConnectionInfo"
+        }
+      ],
+      "required": [
+        "serverName",
+        "port"
+      ]
+    },
+    "MySqlTargetPlatform": {
+      "type": "string",
+      "description": "An enumeration of possible target types when migrating from MySQL",
+      "enum": [
+        "SqlServer",
+        "AzureDbForMySQL"
+      ],
+      "x-ms-enum": {
+        "name": "MySqlTargetPlatformType",
+        "modelAsString": true
+      }
     },
     "ServerLevelPermissionsGroup": {
       "type": "string",
       "description": "Permission group for validations. These groups will run a set of permissions for validating user activity. Select the permission group for the activity that you are performing.",
       "enum": [
         "Default",
-        "MigrationFromSqlServerToAzureDB"
+        "MigrationFromSqlServerToAzureDB",
+        "MigrationFromSqlServerToAzureMI",
+        "MigrationFromMySQLToSQL",
+        "MigrationFromMySQLToAzureDB",
```

Comment:
Is "MigrationFromMySQLToAzureDB" supported? As in MySqlTargetPlatform we only have SqlServer and AzureDbForMySQL